tests/JsonSchemaMapper.Tests/TestTypes.cs

```diff
@@ -303,7 +319,10 @@ public static IEnumerable<ITestData> GetTestDataCore()
             }
             """);
 
+		// would be neat to have the converter expose or be attributed with a schema somehow
```

Comment:
Yeah, it's something we should be exposing in converters once we have a proper exchange type.